src/js/media-controller.js

```diff
@@ -46,6 +47,23 @@ class MediaController extends MediaContainer {
   constructor() {
     super();
 
+    this._volumeUnavailable;
```

Comment:
do you mean the async bug/behavior in mobile Safari? If so, yes. Also, invoking in the constructor to make sure we confirm only where the web component will be instantiated (aka in a browser). We could consider having `_volumeUnavailable` be a module-level variable, since it should be consistent per runtime and across media content, even if we only check it client-side.

Reply:
I think this is related to the comment I just added.

> Also, invoking in the constructor to make sure we confirm only where the web component will be instantiated (aka in a browser)

I'm not totally following here. Is this just related to SSR?